### PR TITLE
Add get port from env

### PR DESCRIPTION
**Pull request by m2web**
### Motivation and Context
Get the port number from the /webapp/.env file

### Description
Instead of hard-coding the port number in the conditional, get the port number from the /webapp/.env file.

### Contribution Checklist

- [ x] The code builds clean without any errors or warnings
- [x ] The PR follows the [Contribution Guidelines](https://github.com/microsoft/chat-copilot/blob/main/CONTRIBUTING.md) and the [pre-submission formatting script](https://github.com/microsoft/chat-copilot/blob/main/CONTRIBUTING.md#development-scripts) raises no violations
- [ ] All unit tests pass, and I have added new tests where possible
- [x ] I didn't break anyone :smile:

### CODE REVIEW

File: webapi/appsettings.Development.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "AIService": { "Type": "OpenAI", "Endpoint": "", "Models": { "Completion": "gpt-3.5-turbo", "Embedding": "text-embedding-ada-002", "Planner": "gpt-3.5-turbo" } }, "Authentication": { "Type": "None", "AzureAd": { "Instance": "", "TenantId": "", "ClientId": "", "Scopes": "access_as_user" } } }
```
Comment (glahaye) [inline webapi/appsettings.Development.json:1]: Did you mean to check this in?

Comment (m2web) [reply webapi/appsettings.Development.json:1]: I don't need to but I saw no sensitive info, so I committed it.

File: scripts/start.sh [hunk 2]
```diff
@@ -7,8 +7,20 @@ set -e
 ScriptDir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 cd "$ScriptDir"
 
+# get REACT_APP_BACKEND_URI from ../webapp/.env
+envContent=$(grep -v '^#' ../webapp/.env | xargs)
+# get the '40443" from REACT_APP_BACKEND_URI=https://localhost:40443/
```
[the remark this answers is not in the document]
Comment (m2web) [reply scripts/start.sh:12]: I noted that after the commit. Do you want me to make another commit with the comment updated?

[the remark this answers is not in the document]
Comment (glahaye) [reply scripts/start.sh:12]: That would be great, yes!

File: webapi/appsettings.Development.json (hunk 1 above)
Comment (glahaye) [reply webapi/appsettings.Development.json:1]: There is no sensitive info, but the file doesn't need to be added either.

Comment (m2web) [reply webapi/appsettings.Development.json:1]: I will remove it in a following commit.

**Comment by teresaqhoang** [comment]
Closing this as a dupe of https://github.com/microsoft/chat-copilot/pull/310

**Pull request closed without merging**